src/commands/gate/README.md

```diff
@@ -1 +1,62 @@
-WORK IN PROGRESS
\ No newline at end of file
+# GATE command
+
+Evaluate Quality Gates rules in Datadog.
+
+## Usage
+
+### Commands
+
+#### `evaluate`
+
+The `evaluate` command evaluates the matching Quality Gates rules in Datadog.
+
+```bash
+datadog-ci gate evaluate [--scope] [--tags] [--dry-run] [--fail-if-unavailable] [--fail-on-empty]
+```
+
+For example:
+
+```bash
+datadog-ci gate evaluate --scope team:backend --scope team:frontend --fail-on-empty
+```
+
+- `--scope` is an array of key value pairs of the form `key:value`. This parameter sets additional scope when retrieving matching rules. Only the rules matching the scope provided will be evaluated.
+- `--tags` is an array of key value pairs of the form `key:value`. This parameter sets global tags that are applied to all results. The upload process merges the tags passed on the command line with the tags in the `DD_TAGS` environment variable. If a key appears in both `--tags` and `DD_TAGS`, the value in `DD_TAGS` takes precedence.
+- `--dry-run` runs the command without the final evaluation step. All other checks are performed. The default value is `false`.
+- `--fail-if-unavailable` fails the command if Datadog is unavailable. The default value is `false`.
+- `--fail-on-empty` fails the command if no matching rules are found in Datadog. The default value is `false`.
+
+### Environment variables
+
+Additionally, you can configure the `gate` command with the following environment variables:
+
+- `DATADOG_API_KEY` or `DD_API_KEY` (**required**): The API key used to authenticate the requests.
+- `DATADOG_APP_KEY` (**required**): The application key used to authenticate the requests.
+- `DD_TAGS`: Sets global tags applied to all spans. The format must be `key1:value1,key2:value2`. The upload process merges the tags passed on the command line with the tags in the `--tags` parameter. If a key appears in both `--tags` and `DD_TAGS`, the value in `DD_TAGS` takes precedence.
+- `DATADOG_SITE`: Your Datadog site, for example, datadoghq.com or datadoghq.eu.
+
+### Optional dependencies
```

Comment:
will this always be an optional dependency?

Reply:
I copied it from Junit and Sarif, which are basically doing the same thing (extracting git-related info from env var). Should I update this?

Reply:
I was just curious whether gating would eventually _require_ git for getting the pertinent pipeline/job/anything. JUnit and Sarif only add this metadata as extra info, AFAIK

Reply:
Oh, that's a good point. Should I mark this as a dependency then (without "optional")?

Reply:
it seems like the safest option, as we can always go to "optional" later on

Reply:
👍 Looks good, changed